day1.md

```diff
@@ -0,0 +1,119 @@
+
+<img src="day1/IMG_20160904_214125.jpg" width="500"><br/>
+Here comes my first <a name="first_meal">meal</a> in Japan! Looking at the picture still makes me hungry now 😂
+
+After resting a bit in the mall, we went to the post office. Man, look at all these postcards! 😱
+<img src="day1/IMG_20160904_224447.jpg" width="500"><br/>
+<img src="day1/IMG_20160904_224508.jpg" width="500"><br/>
+<img src="day1/IMG_20160904_224503.jpg" width="500"><br/>
+They even have series of postcards for trains 😱
+
+Then we headed to <a name="ikebukuro_station">Ikebukuro Station</a> because the place we were staying for the next three nights, booked through [Airbnb]((https://www.airbnb.com/rooms/4585434)), was about 15 mins from the station.
+
+There was a small cheese tart shop with a long line waiting outside at Ikebukuro Station.
+<img src="day1/IMG_20160904_233744.jpg" width="500"><br/>
+<img src="day1/IMG_20160904_233807.jpg" width="500"><br/>
+<img src="day1/14372351_1115746725141687_2298785123460181227_o.jpg" width="500"><br/>
+Hot cheese inside 🔥
+
+On our way to the Airbnb place.<br/>
+<img src="day1/14352021_1115746868475006_8883205870417679098_o.jpg" width="500"><br/>
+<img src="day1/14409899_1115746961808330_4583957709311564211_o.jpg" width="500"><br/>
+
+After checking-in and dropping our bags at our Airbnb place, we were on our way to <a name="odaiba_onsen">Odaiba Onsen</a>, the main event of our first day in Japan!
+<br/><img src="day1/14362487_1115747058474987_3909970043623033642_o.jpg" width="500"><br/>
+
+There was a ~25 mins walk<sup>[4](#walk)</sup> from Teleport Station to Odaiba Onsen. It felt like a pretty long walk for us, mainly because we were tired out by the hot weather 🔥. We even had a small break near the end of the walk and got some foods at a Family Mart, one of the three most common convenient stores in Japan.
+
+We were pretty excited when we finally arrived and gonna spend the rest of the day in door!
+<br/><img src="day1/14434912_1115747191808307_2336258069687317341_o.jpg" width="500"><br/>
+
+The first thing we needed to do, after checking-in, was to figure out our sizes and styles for Yukatas. We were gonna wear them the whole time hanging in Odaiba Onsen.
+
+Odaiba Onsen was huge inside! Here's the [floor map](https://jonellepatrick.com/oedo-onsen-2/).
+<br/><img src="day1/oedoonsenmap.jpg" width="500"><br/>
+
+We started by hanging out in "Town Square":
+>[which has been built to resemble an Edo-era town. Lining the “street” are many shops, food stands and restaurants – udon and soba noodles, sushi, unagi (eel), oden, beer, sake, etc.](https://jonellepatrick.com/oedo-onsen-2/)
+
+<br/><img src="day1/IMG_20160905_161521.jpg" width="500">
+<br/><img src="day1/IMG_20160905_200120.jpg" width="500"><br/>
+
+Matt and Katherine got crepes and I got a huge takoyaki:
```

Comment:
I don't know how much of a grammar Nazi you want me to be, so lemme know how much you care. Technically, there should be a comma before the "and" since it is joining two independent clauses.